jujugui/static/gui/src/app/components/shared/status/table/test-table.js

```diff
@@ -0,0 +1,39 @@
+/* Copyright (C) 2018 Canonical Ltd. */
+'use strict';
+
+const React = require('react');
+const enzyme = require('enzyme');
+
+const StatusTable = require('./table');
+
+describe('StatusTable', () => {
```

Comment:
The two utility methods in this component are not tested as they are just passed through to the `BasicTable` could you add some unit tests for those plz.